docs/excel/excel-add-ins-comments.md

```diff
@@ -197,6 +197,105 @@ Excel.run(function (context) {
 });
 ```
 
+## Comment events
```

Comment:
How do we reconcile this section with the [Excel events](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-events) article? At minimum, we should link to it to help demonstrate the patterns.

Reply:
After reviewing the Events article, I think linking to it from this article section would be helpful. I'm also considering adding a line to the Events article that links to this Comment events article section.